pom.xml

```diff
@@ -65,7 +65,7 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <mavenVersion>3.1.1</mavenVersion>
+    <mavenVersion>3.3.1</mavenVersion>
```

Comment:
This upgrade should be separate ticket.

Reply:
I don't think it's needed anymore.  I'll check if this can be removed from this PR.

Reply:
When was `${maven.multiModuleProjectDirectory}` introduced? 3.1.1 or 3.3.1?

Reply:
In 3.3.1 ...

Reply:
The you need to do the upgrade first or document that this option requires Maven 3.3.1 and leave 3.1.1 as baseline.

Reply:
I've changed the requirements back to 3.1.1, even though the fast option won't be honoured if not running on 3.3.1 (or at least without the `${maven.multiModuleProjectDirectory}` property.